client/my-sites/google-my-business/style.scss

```diff
@@ -1 +1,13 @@
-@import './select-business-type/style';
+.google-my-business {
+	h1 {
```

Comment:
Do we need an `h1` rule specific to google my business?
It seems a bit strange that we have to define specific styling rules for this project. It would make more sense to have more global rules regarding titles and reuse them everywhere in calypso.

Reply:
I see your point, and that's a valid concern, but I don't think we have a global rule for titles (at least I couldn't find any). We could define one inside the `Card` component but then I'm worried of potential  side-effects.

Reply:
Two points:

1. We should use a class as a selector, not an element name, as the coding guidelines specify: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md?term=css

2. We should create a design pattern for these headers - I'll do that soon :)